tests/crew_test.py

```
@@ -1198,9 +1370,81 @@ def test_crew_inputs_interpolate_both_agents_and_tasks_diff():
+    )
+
+    task = Task(
+        description="Give me an analysis around {topic}.",
+        expected_output="{points} bullet points about {topic}.",
+        agent=agent,
+    )
+
+    crew = Crew(agents=[agent], tasks=[task])
+
+    with patch.object(Agent, "interpolate_inputs") as interpolate_agent_inputs:
+        with patch.object(Task, "interpolate_inputs") as interpolate_task_inputs:
```

Ruff

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)